mondrian/src/main/java/mondrian/olap/Mondrian.xml

```diff
@@ -966,7 +966,7 @@ Revision is $Id$
         <Attribute name="datatype" required="false">
             <Doc>
                 The datatype of this measure:
-                String, Numeric, Integer, Boolean, Date, Time or Timestamp.
+                String, Numeric, Integer.
 
```

Comment:
Thanks Andrei.  Those datatypes wouldn't make sense as sums, but seem useful to me with other aggregators-- e.g. count of dates, minimum time, etc.  

type="Boolean" works for me with a hand written schema.  Would be worth trying the other datatypes, too.

I looked around for the unit test that Will G. mentioned in MONDRIAN-707 to get some context on why he thought they should not be supported, but couldn't find it.  Did you?